packages/tools/pie-icons-webc/build.js

```diff
@@ -38,11 +38,10 @@ const componentSelector = '${kebabCase(name)}';
 export class ${name} extends LitElement implements IconProps {
     // The following styles make sure that the icon will be sized correctly
     static styles = css\`
-        :host-context(pie-icon-button) svg,
-        :host-context(pie-button) svg {
-            display: block;
-            width: var(--btn-icon-size);
-            height: var(--btn-icon-size);
+        :host svg {
```

Comment:
This change will mess up icon size for all our icons making the svg compainer 24px X 24px. Here is a Link component with an icon as an example:
<img width="156" alt="Screenshot 2023-12-21 at 10 49 26" src="https://github.com/justeattakeaway/pie/assets/19548183/3d32a7d1-8e03-4809-b3cf-35c017b34cf0">

Reply:
I created a PR for the fix: https://github.com/justeattakeaway/pie/pull/1127